src/SourceLocator/SourceStubber/PhpStormStubsMap.php

```diff
@@ -0,0 +1,4910 @@
+    'imagick/imagick.php',
+    'imap/imap.php',
+    'inotify/inotify.php',
+    'interbase/interbase.php',
+    'intl/IntlChar.php',
+    'intl/intl.php',
+    'json/json.php',
+    'judy/judy.php',
+    'ldap/ldap.php',
+    'libevent/libevent.php',
+    'libsodium/libsodium.php',
+    'libxml/libxml.php',
+    'lua/lua.php',
+    'mailparse/mailparse.php',
+    'mapscript/mapscript.php',
+    'mbstring/mbstring.php',
+    'mcrypt/mcrypt.php',
+    'memcache/memcache.php',
+    'memcached/memcached.php',
+    'ming/ming.php',
+    'mongo/mongo.php',
+    'mongodb/mongodb.php',
+    'mqseries/mqseries.php',
+    'msgpack/msgpack.php',
+    'mssql/mssql.php',
+    'mysql/mysql.php',
+    'mysql_xdevapi/mysql_xdevapi.php',
+    'mysqli/mysqli.php',
+    'ncurses/ncurses.php',
+    'newrelic/newrelic.php',
+    'oauth/oauth.php',
+    'oci8/oci8.php',
+    'odbc/odbc.php',
+    'openssl/openssl.php',
+    'password/password.php',
+    'pcntl/pcntl.php',
+    'pcre/pcre.php',
+    'pdflib/PDFlib.php',
+    'pdo_ibm/pdo_ibm.php',
+    'pgsql/pgsql.php',
+    'posix/posix.php',
+    'pspell/pspell.php',
+    'pthreads/pthreads.php',
+    'radius/radius.php',
+    'rar/rar.php',
+    'rdkafka/RdKafka.php',
+    'rdkafka/functions.php',
+    'readline/readline.php',
+    'recode/recode.php',
+    'redis/Redis.php',
+    'redis/RedisCluster.php',
+    'regex/ereg.php',
+    'rrd/rrd.php',
+    'session/SessionHandler.php',
+    'session/session.php',
+    'shmop/shmop.php',
+    'snmp/snmp.php',
+    'soap/soap.php',
+    'sockets/sockets.php',
+    'sodium/sodium.php',
+    'solr/SolrClient.php',
+    'solr/functions.php',
+    'sqlite3/sqlite3.php',
+    'sqlsrv/sqlsrv.php',
+    'ssh2/ssh2.php',
+    'standard/_standard_manual.php',
+    'standard/_types.php',
+    'standard/basic.php',
+    'standard/standard_0.php',
+    'standard/standard_1.php',
+    'standard/standard_2.php',
+    'standard/standard_3.php',
+    'standard/standard_4.php',
+    'standard/standard_5.php',
+    'standard/standard_6.php',
+    'standard/standard_7.php',
+    'standard/standard_8.php',
+    'standard/standard_9.php',
+    'suhosin/suhosin.php',
+    'svn/svn.php',
+    'sybase/sybase_ct.php',
+    'sync/sync.php',
+    'sysvmsg/sysvmsg.php',
+    'sysvsem/sysvsem.php',
+    'sysvshm/sysvshm.php',
+    'tidy/tidy.php',
+    'tokenizer/tokenizer.php',
+    'uopz/uopz.php',
+    'v8js/v8js.php',
+    'wddx/wddx.php',
+    'win32service/win32service.php',
+    'wincache/wincache.php',
+    'xcache/xcache.php',
+    'xdebug/xdebug.php',
+    'xhprof/xhprof.php',
+    'xml/xml.php',
+    'xmlreader/xmlreader.php',
+    'xmlrpc/xmlrpc.php',
+    'xmlwriter/xmlwriter.php',
+    'xsl/xsl.php',
+    'yaf/yaf.php',
+    'yaf/yaf_namespace.php',
+    'yaml/yaml.php',
+    'yar/yar.php',
+    'zend/zend.php',
+    'zend/zend_f.php',
+    'zip/zip.php',
+    'zlib/zlib.php',
+    'zmq/zmq.php',
+];
+
+return [
+    [
+        'AMQPBasicProperties' => &$files[28],
```

Comment:
Uhhh, what is going on here? Why is this by-ref?

Reply:
Indeed: this should be a constant expression, as that would allow some degree of OpCache optimisations. Memory optimisation shouldn't be a big concern for such a small definition file

Reply:
Done.